Makefile

```diff
@@ -30,10 +30,17 @@ build: ## build files but do not run a server
 	${JEKYLL} build
 .PHONY: build
 
-check: build ## validate HTML
-	yamllint .
+check-html: build ## validate HTML
 	timeout 120s bundle exec htmlproofer --http-status-ignore 405,999 --url-ignore "/.*localhost.*/","/.*vimeo\.com.*/" --file-ignore "/.*\/files\/.*/" ./_site
```

Comment:
Can we use something else than the timeout? It is not working on OSX

Reply:
sure thing, I will look into alternatives

Reply:
I removed the `timeout` again, because by splitting it into separate jobs, nothing needs to run after this anymore so it should be ok if it just hits the travis timeout when it hangs I think? (@bgruening I assume thats why you added this timeout? or was there another reason?)

Reply:
The timeout was needed because the testing generated no output for 10min, which results in a travis error. If this is not the case anymore it can be removed. But travis fails if 10min no output.